lib/matplotlib/rcsetup.py

```diff
@@ -147,6 +147,14 @@ def validate_bool(b):
         raise ValueError('Could not convert "%s" to bool' % b)
 
 
+def validate_color_or_bool(c):
+    """Convert c to color (preferentially) or bool"""
+    try:
+        return validate_color(c)
+    except ValueError:
+        return validate_bool(c)
```

Comment:
```suggestion
        return validate_bool(c)
    except ValueError:
        raise ValueError(f'Could not convert "{c}" to color or bool')
```
Otherwise you get the somewhat misleading ValueError message from the bool converter ('Could not convert "palered" to bool').

Reply:
Do you mean something like this?
```
try: validate_color(c)
except: pass
try: validate_bool(c)
except: raise error
```

Reply:
Yes. Sorry, the code suggestion was not correct.

Reply:
No problem, just verifying that what I wrote was right before pushing it.